controllers/storageclient_controller.go

```diff
@@ -227,6 +233,12 @@ func (r *StorageClientReconciler) reconcilePhases() (ctrl.Result, error) {
 			return reconcile.Result{}, fmt.Errorf("failed to update StorageClient with claim processed annotation: %v", err)
 		}
 	}
+	// reconcile  remote noobaa only if it is not a local client
+	if _, exists := r.storageClient.GetAnnotations()["is-local-client"]; !exists {
+		if err := r.reconcileRemoteNoobaa(); err != nil {
+			return reconcile.Result{}, err
+		}
+	}
```

Comment:
Please move the reconciliation of noobaa above the code reconciling the default storage classes. We kept them last for a reason.

Reply:
moved